### PR TITLE
DRDT-54: In-content nl module

### PR DESCRIPTION
**Pull request by ChrisHursty**

### CODE REVIEW

**Comment by fariasf** [comment]
Created new staging environment for pr-82. Here are some useful links:

[WordPress Admin](https://pr-82-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-82-drdt.pantheonsite.io/)
[Article](https://pr-82-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-82-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-82-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-82-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2750/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-82 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [90](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/pr-82/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✘ |
| Article | [71](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/pr-82/article/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✘ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/pr-82/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/pr-82/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/pr-82/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2751/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

File: web/wp-content/themes/bumblebee/sass/site/primary/global.scss [hunk 1]
```diff
@@ -156,6 +156,53 @@ img {
...
+
+		input {
+			border-radius: 0;
+			font-size: 20px;
+			padding: 10px;
+			width: 70%;
+		}
+
+		button {
+			background-color: #ffc10e;
```
Comment (ChrisHursty) [inline web/wp-content/themes/bumblebee/sass/site/primary/global.scss:189]: TODO: https://readersdigest.atlassian.net/browse/DRDT-112

File: web/wp-content/themes/bumblebee/sass/site/primary/global.scss [hunk 2]
```diff
@@ -156,6 +156,53 @@ img {
...
+			font-size: 17px;
+			padding: 0;
+			text-transform: uppercase;
+			transition: background .3s;
+			width: 30%;
+		}
+	}
+
+	.diyu-logo {
+		background-color: #999;
```
Comment (ChrisHursty) [inline web/wp-content/themes/bumblebee/sass/site/primary/global.scss:201]: TODO: https://readersdigest.atlassian.net/browse/DRDT-112

File: web/wp-content/themes/bumblebee/footer.php [hunk 3]
```diff
@@ -78,18 +78,7 @@
...
-						<input type="text" id="email" placeholder="Email Address"></input>
-						<button type="submit" id="subscribe">Sign Up</button>
-					</form>
-				</div>
-				<div class="diyu-logo">
-					<a data-analytics-metrics='{"name":"Subscribe link","module":"footer","position":"magazine subscription"}' href="<?php echo esc_html( get_theme_mod( 'bumblebee_footer_nl_subscribe_url' ) ); ?>" target="_blank" rel="noopener noreferrer">
-						<img src="<?php echo esc_html( get_theme_mod( 'bumblebee_footer_nl_subscribe_image' ) ); ?>" alt="" style="width:<?php echo esc_html( get_theme_mod( 'bumblebee_footer_nl_subscribe_image_width' ) ); ?>px"></img>
-					</a>
-				</div>
+				<?php newsletter_module(); ?>
```
Comment (fariasf) [inline web/wp-content/themes/bumblebee/footer.php:81]: Is it worth having a ticket to make this a widget?

File: web/wp-content/themes/bumblebee/functions.php [hunk 4]
```diff
@@ -444,3 +444,34 @@ function enable_svg_upload( $existing_mimes = array() ) {
 	return $existing_mimes;
 }
 add_filter( 'upload_mimes', 'enable_svg_upload' );
+
+/**
+ * Add In Content NL Module AFTER the_content
+ *
+ * @param  content $content Append NL to the_content
+ */
+function newsletter_after_the_content( $content ) {
```
Comment (fariasf) [inline web/wp-content/themes/bumblebee/functions.php:453]: This function seems to be unused, do we need to keep it?

File: web/wp-content/themes/bumblebee/single.php [hunk 5]
```diff
@@ -130,6 +130,13 @@ function () {
 					</aside>
 				</div>
 			</div>
+			<div class="pure-u-md-3-24 pure-u-lg-3-24 pure-u-xl-3-24 hide-on-mobile"></div>
```
Comment (fariasf) [inline web/wp-content/themes/bumblebee/single.php:133]: What is this empty div for?

**Comment by fariasf** [comment]
Updated staging environment for pr-82. Here are some useful links:

[WordPress Admin](https://pr-82-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-82-drdt.pantheonsite.io/)
[Article](https://pr-82-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-82-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-82-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-82-drdt.pantheonsite.io/jobsite-tips/)

File: web/wp-content/themes/bumblebee/single.php (hunk 5 above)
Comment (ChrisHursty) [reply web/wp-content/themes/bumblebee/single.php:133]: It's the (empty) left sidebar, I was following on from how the template was coded

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2757/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-82 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [95](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/pr-82/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/pr-82/article/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/pr-82/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/pr-82/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/pr-82/hub/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2756/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-82. Here are some useful links:

[WordPress Admin](https://pr-82-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-82-drdt.pantheonsite.io/)
[Article](https://pr-82-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-82-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-82-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-82-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2763/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-82 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [95](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/pr-82/homepage/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/pr-82/article/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/pr-82/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/pr-82/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/pr-82/hub/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2764/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-82. Here are some useful links:

[WordPress Admin](https://pr-82-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-82-drdt.pantheonsite.io/)
[Article](https://pr-82-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-82-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-82-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-82-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2805/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-82 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [95](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/pr-82/homepage/lighthouse.report.html) | [94](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [97](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/pr-82/article/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/pr-82/listicle/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/pr-82/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [59](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/pr-82/hub/lighthouse.report.html) | [93](https://circleci.com/gh/fariasf/drdt-test/2806/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✘ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-82. Here are some useful links:

[WordPress Admin](https://pr-82-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-82-drdt.pantheonsite.io/)
[Article](https://pr-82-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-82-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-82-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-82-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2811/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
⚠ Frontend performance tests failed!

Full Lighthouse performance results:

| Page/Branch | pr-82 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [87](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/pr-82/homepage/lighthouse.report.html) | [96](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✘ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/pr-82/article/lighthouse.report.html) | [97](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [97](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/pr-82/listicle/lighthouse.report.html) | [52](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/pr-82/page/lighthouse.report.html) | [98](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [61](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/pr-82/hub/lighthouse.report.html) | [92](https://circleci.com/gh/fariasf/drdt-test/2812/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✘ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-82. Here are some useful links:

[WordPress Admin](https://pr-82-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-82-drdt.pantheonsite.io/)
[Article](https://pr-82-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-82-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-82-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-82-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2834/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-82 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [94](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/pr-82/homepage/lighthouse.report.html) | [57](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [96](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/pr-82/article/lighthouse.report.html) | [63](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/pr-82/listicle/lighthouse.report.html) | [69](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/pr-82/page/lighthouse.report.html) | [59](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/pr-82/hub/lighthouse.report.html) | [59](https://circleci.com/gh/fariasf/drdt-test/2833/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

**Comment by fariasf** [comment]
Updated staging environment for pr-82. Here are some useful links:

[WordPress Admin](https://pr-82-drdt.pantheonsite.io/wp/wp-admin)

[Homepage](https://pr-82-drdt.pantheonsite.io/)
[Article](https://pr-82-drdt.pantheonsite.io/jobsite-tips/abandoned-cable-removal/)
[Listicle](https://pr-82-drdt.pantheonsite.io/jobsite-tips/exterior/12-scaffolding-safety-tips-and-handy-hints/)
[Page](https://pr-82-drdt.pantheonsite.io/about-construction-pro-tips/)
[Hub](https://pr-82-drdt.pantheonsite.io/jobsite-tips/)

**Comment by fariasf** [comment]
There are visual regressions! 

View the [full visual regression test report](https://circleci.com/gh/fariasf/drdt-test/2868/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by fariasf** [comment]
Congrats! Frontend performance tests passed!

Full Lighthouse performance results:

| Page/Branch | pr-82 | master | Result |
| --- | ---: | ---: | :---: |
| Homepage | [84](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/pr-82/homepage/lighthouse.report.html) | [57](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/master/homepage/lighthouse.report.html) | ✔ |
| Article | [98](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/pr-82/article/lighthouse.report.html) | [63](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/master/article/lighthouse.report.html) | ✔ |
| Listicle | [98](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/pr-82/listicle/lighthouse.report.html) | [68](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/master/listicle/lighthouse.report.html) | ✔ |
| Page | [98](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/pr-82/page/lighthouse.report.html) | [62](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/master/page/lighthouse.report.html) | ✔ |
| Hub | [97](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/pr-82/hub/lighthouse.report.html) | [57](https://circleci.com/gh/fariasf/drdt-test/2867/artifacts/0/artifacts/lighthouse_results/master/hub/lighthouse.report.html) | ✔ |


(**Note: Tests are against the _NOADS_ version of the pages!**)

File: web/wp-content/themes/bumblebee/archive.php [hunk 6]
```diff
@@ -49,31 +49,77 @@ function () {
...
+						'tf'       => $tf_slot,
+						'pos'      => $slot_name,
+						'location' => $slot_name,
+					],
+				]
+			);
+			?>
 		</section>
-		<section class="full-width-ad">
+		<div class="pure-g recipes">
```
Comment (fariasf) [inline web/wp-content/themes/bumblebee/archive.php:112]: I know the `recipes` class was here before... but why? CPT doesn't have recipes.

File: web/wp-content/themes/bumblebee/functions.php (hunk 4 above)
Comment (ChrisHursty) [reply web/wp-content/themes/bumblebee/functions.php:453]: Created ticket https://readersdigest.atlassian.net/browse/DRDT-118 to address such issues

File: web/wp-content/themes/bumblebee/footer.php (hunk 3 above)
Comment (ChrisHursty) [reply web/wp-content/themes/bumblebee/footer.php:81]: This was added to the customizer in https://github.com/fariasf/drdt-test/pull/82/

File: web/wp-content/themes/bumblebee/archive.php (hunk 6 above)
Comment (ChrisHursty) [reply web/wp-content/themes/bumblebee/archive.php:112]: Created ticket https://readersdigest.atlassian.net/browse/DRDT-118 to address such issues

**Pull request merged**